src/sentry/api/endpoints/monitor_checkin_details.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import absolute_import
```

Comment:
no tests for these endpoints yet - so probably should get some basic "does it work at all" coverage in first